content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md

```diff
@@ -0,0 +1,481 @@
+      battery = BatteryValue;
+    }
+
+    // check if the value of the characteristic has been updated
+    if (alertLevel.valueUpdated()) {
+
+      alertLevel.readValue(AlertValue);
+
+      Serial.print("Alert: ");
+      Serial.println(AlertValue);
+
+      
+      if (AlertValue == 1) {    // if the Alert = 1 means the door was opened
+        control = !control;     // as variables are updated if they change, add a simple dot "." to be able to update it if happens twice consecutive.
+        if (control) {
+          DoorEvent = "Door opened";
+        } else {
+          DoorEvent = "Door opened.";
+        }
+
+        alertStatus = false;
+
+      } else if (AlertValue == 2) {   // if the Alert = 2 means the door was forced
+        control = !control;           // as variables are updated if they change, add a simple dot "." to be able to update it if happens twice consecutive.
+        if (control) {
+          DoorEvent = "Intruder detected!";
+        } else {
+          DoorEvent = "Intruder detected!.";
+        }
+
+        alertStatus = true;
+      }
+    }
+
+    // turn off green LED if disconnected from Cloud.
+    if (ArduinoCloud.connected()) {
+      digitalWrite(LEDG, LOW);
+    } else {
+      digitalWrite(LEDG, HIGH);
+    }
+
+    ArduinoCloud.update();
+  }
+```
+### The Cloud Dashboard
+
+Taking advantage of the Arduino Cloud, we can seamlessly integrate a simple but powerful dashboard to monitor and visualize the status of the system in real-time:
+
+![Arduino Cloud project dashboard ](assets/dashboard.png)
+
+Within the Arduino Cloud's dashboard, the system variables can be monitored. We have a battery level indicator widget accompanied by a time series graph, a chat-looking widget to store the events historically, a red LED to shine when an intruder is detected by the system, and a green LED to show the Bluetooth® Low Energy connection status between the Nicla Voice and the Portenta H7 host. 
+
+We can easily access this dashboard from a PC, mobile phone or tablet from anywhere, receiving an instantaneous update wherever we are. In addition, we can set different integrations to complement our project, for example, setting up an IFTTT automation to receive an email, a cellphone notification, or even triggering some automation like turning on all the lights at home whenever an alert is fired.
+
+![Door opening event](assets/opened_door.png)
+
+![Intruder detected event](assets/forced_door.png)
+
+## Full Intruder Detector Example
+
+
+All the necessary files to replicate this application notes can be found below:
+
+* The complete code can be downloaded [here](assets/Nicla_Portenta_Codes.zip)
+* The enclosure 3D printed models can be downloaded [here](assets/Intruder_Detector_3D_Files.zip). 
+* The Edge Impulse project is public [here](https://mltools.arduino.cc/public/204744/latest) so you can clone it and modify it to adapt it to your needs by improving the dataset or model architecture for a custom deployment.
```

Comment:
The Machine Learning Tools project instead of Edge Impulse

Reply:
done